doc/contributor/workflow.md

```diff
@@ -63,16 +63,20 @@ $ echo MX_BINARY_SUITES=truffle,sdk > mx.truffleruby/env
 $ jt build
 ```
 
-By default the `jt build` command only builds the JVM-based launcher for TruffleRuby.
-If you'd like to build the native image using the Substrate VM, you can do so by
-providing an extra argument to the build command.
+By default the `jt build` command only builds the JVM-based launcher for
+TruffleRuby. If you'd like to build the native image using the Substrate VM, you
+can do so by providing an extra argument to the build command.
 
 ```bash
 $ jt build native
 ```
 
 This will create a new executable named `native-ruby` in the `bin/` directory.
 
+Note that build information such as the date and Git revision hash will not be
+updated when you build for a second time. Releases should always be built from
+scratch.
```

Comment:
Why not? I think with `jt build` the annotation processors run every time, no?
Or maybe `PopulateBuildInformation` needs to be modified to re-run it?

Reply:
It won't look at files which have already been compiled if the file hasn't changed and the annotation processors haven't changed.